sidebars.js

```diff
@@ -541,82 +545,7 @@ module.exports = {
       collapsed: false,
       link: {
         type: "doc",
-        id: "concepts/index",
```

Comment:
Somehow the Concepts sidebar was duplicated in the sidebar.js file and the second one was overriding the first.